test/ibm/test_random.py

```diff
@@ -54,6 +55,7 @@ def can_access_extractor(self):
         except IBMError:
             return False
 
+    @skipIf(terra_version <= '0.2.2', "Need Terra > 0.2.2")
```

Comment:
We should skip if `qiskit_rng` version is <=0.2.2, not `terra` version...

Reply:
oops 😅